packages/ui/v2/core/Shell.tsx

```diff
@@ -428,17 +413,54 @@ const navigation: NavigationItemType[] = [
       },
     ],
   },
+  {
+    name: "more",
+    href: "/more",
+    icon: Icon.FiMoreHorizontal,
+  },
+  {
+    name: "Routing Forms",
+    href: "/apps/routing_forms/forms",
+    icon: Icon.FiFileText,
+    isCurrent: ({ router }) => {
+      return router.asPath.startsWith("/apps/routing_forms/");
+    },
+  },
+  {
+    name: "workflows",
+    href: "/workflows",
+    icon: Icon.FiZap,
+  },
   {
     name: "settings",
     href: "/settings",
     icon: Icon.FiSettings,
   },
 ];
 
+const MORE_NAME = "more";
+const moreSeparatorIndex = navigation.findIndex((item) => item.name === MORE_NAME);
+const desktopNavigationItems: NavigationItemType[] = [];
+const mobileNavigationBottomItems: NavigationItemType[] = [];
+const mobileNavigationMoreItems: NavigationItemType[] = [];
+navigation.forEach((item, index) => {
```

Comment:
one for each to catch'em all @zomars

Reply:
@alannnc I take your forEach and up it with reduce

![Simpsons Monkey Fight GIF](https://media.tenor.com/images/a7fb535c2d1afab8294d265f37c70363/tenor.gif)